lib/afinter.h

```diff
@@ -28,6 +28,13 @@
 #include "driver.h"
 #include "logsource.h"
 
+enum
+{
+  AF_LIVE_COLLECTION_STARTED,
```

Comment:
These constants should be named in a consistent manner. The AF part is not really necessary, however they should have a consistent prefix, like LC_, LIVE_COLLECTION_ or something similar.
That way the constants would become something like LIVE_COLLECTION_STARTED, LIVE_COLLECTION_ALREADY_RUNNING and LIVE_COLLECTION_INTERNAL_SRC_ALREADY_IN_USE

Reply:
Note: The `LC_` prefix is reserved (by `CfgTree`: `LC_CATCHALL`, `LC_FALLBACK`, etc.).
`AFINTER_`, for example, can be a good choice.

Reply:
@MrAnno : Yes, I am now using ```AINTER_```